### ISSUE TITLE
Update navigation demos to show accessible experience

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#5260]
**Describe the issue. What is the expected and unexpected behavior?**
Currently when you navigate to the navigation demos by keyboard, you are unable to navigate to the side panel and click other links. This is likely due to the fact that the links do not have href's. However, when href's are added and the user tabs through the page, keyboard focus disappears off the page. You can determine from the link previews at the bottom left that focus is still going to these links even when the panel is closed. I believe that we could fix this by altering the items to display none when the panel is collapsed. However, we need to account for transitioned events so that if we ever add transitions, the panel items don't just disappear before the transition.

This issue was first noticed in OpenShift. When comparing it to the react demos, we noticed that our demos have the same issue. It would make sense to solve this issue in the context of PF first.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

Open [any navigation demo](https://www.patternfly.org/v4/components/navigation/react-demos/default-nav/) and attempt to navigate by keyboard. 

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug
